minnie-kenny.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/sh
```

Comment:
from https://minnie-kenny.readthedocs.io/en/latest/Install/ -- does not require review